### PR TITLE
chore(angular): Upgrade to Angular 10

### PR DESCRIPTION
**Pull request by arjenbrandenburgh**

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -2,6 +2,19 @@
   "extends": ["tslint:recommended"],
   "rulesDirectory": ["node_modules/codelyzer"],
   "rules": {
+    "align": {
```
Comment (devoto13) [inline tslint.json:5]: Almost all changes in this file are formatting rules and formatting is handled by Prettier in this repository, so you can revert all of them.

**Comment by devoto13** [comment]
Also, please pick up `yarn.lock` from my PR as I did some cleanup to get rid of `yarn check` warnings.

File: tslint.json (hunk 1 above)
Comment (arjenbrandenburgh) [reply tslint.json]: Reverted this file

File: package.json [hunk 2]
```diff
@@ -43,47 +43,48 @@
...
+    "karma-jasmine": "~3.3.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^10.0.0",
     "prettier": "2.0.2",
-    "protractor": "~5.4.3",
+    "protractor": "~7.0.0",
     "rxjs": "^6.5.4",
     "ts-node": "~8.8.1",
-    "tslib": "^1.11.1",
+    "tslib": "^2.0.0",
```
Comment (devoto13) [inline package.json:85]: Please also move `tslib` to [dependencies](https://github.com/FortAwesome/angular-fontawesome/pull/258/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R53) as this is now the recommended way.

**Pull request merged**